docs/book/content/theory/households.md

```diff
@@ -77,9 +77,14 @@ fig = pp.plot_elliptical_u(p)
 glue("labor_utility", fig, display=False)
 ```
 
-```{glue:figure} labor_utility
```

Comment:
@prrathi This figure was supposed to be generated from the code above.  Did it render for you when you built the document locally? 

I have no problem generating the figure locally, but it doesn't seem to render when built with the GH Action.

Reply:
@jdebacker Yes, originally it wasn't but after the changes the figure rendered on the local html.

Reply:
That figure plots fine locally on my machine using @prrathi's branch.
<img width="1409" alt="HHellip_screenshot" src="https://user-images.githubusercontent.com/4317690/92034209-26961200-ed2a-11ea-8f0d-4025f685d9b6.png">

Reply:
That's the png that he's inserted that you are seeing - it's not the plot from OG-USA, which will look like:
<img width="1005" alt="Screen Shot 2020-09-02 at 6 15 41 PM" src="https://user-images.githubusercontent.com/10715924/92042502-59e79980-ed48-11ea-8672-04d262c84964.png">

Reply:
@jdebacker you're right. @prrathi's code is the following, which is what produced my plot above.
````python
+++
```{code-cell} ogusa-dev
:tags: [hide-cell]
from myst_nb import glue
import ogusa.parameter_plots as pp
from ogusa import Specifications
p = Specifications()
p.update_specifications({'frisch': 0.6})
fig = pp.plot_elliptical_u(p)
glue("labor_utility", fig, display=False)
```

```{figure} ./images/EllipVsCFE_MargUtil.png
---
height: 500px
name: FigMDUcompar
---



Comparison of CFE marginal disutility of leisure $\theta=1.67$ to fitted elliptical utility
```
````

When I use the code that is currently in OG-USA `households.md`, I get no figure.
````python
+++
```{code-cell} ogusa-dev
:tags: [hide-cell]
from myst_nb import glue
import ogusa.parameter_plots as pp
from ogusa import Specifications
p = Specifications()
p.update_specifications({'frisch': 0.6})
fig = pp.plot_elliptical_u(p)
glue("labor_utility", fig, display=False)
```

```{glue:figure} labor_utility
:figwidth: 750px
:name: "FigMDUcompar"

Comparison of CFE marginal disutility of leisure $\theta=0.6$ to fitted elliptical utility
```
````
<img width="1371" alt="HHellip_screenshot2" src="https://user-images.githubusercontent.com/4317690/92044488-96f95f00-ed3b-11ea-8c7d-0246d948ed7d.png">

I don't understand why the `{glue:figure} directive is not working as described in the [`jupyter-book` documentation](https://jupyterbook.org/content/code-outputs.html#the-glue-figure-directive).

Reply:
I think that if we can't get the `{glue:figure}` directive to work, then we just use @prrathi's workaround of taking a separate figure from an already-created image .png.

Reply:
@rickecon I agree.  I had thought it was an issue with the GH Action, but that you can't replicate locally, I wonder if I have some package in my env that is missing in yours (and that the GH Action is run in).

I'll try to figure that out, but I agree that we can just revert to the cached image for the time being.

Reply:
Also - it's funny that the graph doesn't show up because tables that use the `glue` function seem to be showing up on deployment (and I assume on your local build?).